accounts/abi/type.go

```diff
@@ -57,16 +58,16 @@ var (
 	// Types can be in the format of:
 	//
 	// 	Input  = Type [ "[" [ Number ] "]" ] Name .
-	// 	Type   = [ "u" ] "int" [ Number ] .
+	// 	Type   = [ "u" ] "int" [ Number ] [ x ] [ Number ].
```

Comment:
If there's an `x` included, it's always proceeded by a number, not? If yes, perhaps it's cleaner to write `[ "x" Number ]` as a single optional part, not two.